src/views/AccountDetails.vue

```diff
@@ -0,0 +1,409 @@
+<template>
+    <div class="card-content">
+      <div class="media">
+        <div class="media-content">
+          <div class="columns">
+            <div class="column">
+            </div>
+            <div class="column">
+              <div class="field is-grouped is-pulled-right">
+                <p class="control">
+                  <button class="button is-small save-document-button"
+                          title="Save changes"
+                          data-qa="save-changes"
+                          :disabled="parseInt(userDetails.acl) > 20">
+                    <span class="icon is-small">
+                      <i class="fas fa-save"></i>
+                    </span>
+                    <span> Save </span>
+                  </button>
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <hr/>
+      <div v-show="!loadingDocCounter && selectedBranch && accountId">
+        <div class="content collapsible-content px-5 py-5">
+          <div class="columns columns-divided">
+            <div class="column is-4">
+              <div class="field">
+                <label class="label is-small">
+                  Your Name
+                </label>
+                <div class="control suffix">
```

Comment:
The input doesn't have a suffix

```suggestion
                <div class="control">
```

Reply:
Removed, done.